app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java

```diff
@@ -1019,6 +1025,25 @@ public LatLng evaluate(float fraction, LatLng startValue, LatLng endValue) {
         }
     }
 
+    public void setMapCenter(Place place) {
+        mapView.getMapAsync(mapboxMap1 -> {
```

Comment:
It seems like this method does not just sets a variable, it centers the map instead. Maybe a name like centerMapToMarker or centerMapToPlace could be a better naming. 

And please add javadocs for this method too.